### ISSUE TITLE
Why still in beta version?

### ISSUE THREAD

**Issue by mbugni** [flathub/org.kde.skanlite#18]
I tested this app and it works fine.
It works just like [SimpleScan](https://flathub.org/apps/org.gnome.SimpleScan).
Is there any blocking issue that is preventing to publish it on Flathub?

**Comment by travier** [comment]
From https://invent.kde.org/teams/flathub/issues/-/issues/13, which points to https://github.com/flathub/flathub/pull/2871#issuecomment-1065230294:

> I can confirm that invent.kde.org/packaging/flatpak-kde-applications/-/blob/master/org.kde.skanpage.json & invent.kde.org/packaging/flatpak-kde-applications/-/blob/master/org.kde.skanlite.json work with an USB scanner but this one doesn't so it is probably missing something.

**Comment by mbugni** [comment]
Just to understand: Skanpage and Skanlite native apps work with an usb-scanner, but Flatpak ones don't. Isn't it?

**Comment by travier** [comment]
From memory, the Flatpak builds from KDE development versions worked but not the Flathub ones. This was back when we used binary factory (Jenkins), we are using now GitLab CI and it would need to be tested again.

**Comment by mbugni** [comment]
I'm not sure I understand. It seems that a Flatpak build works but another Flatpak build doesn't.
It sounds strange to me. I think all Flatpak build should produce the same result.

Anyway, I tested [latest build](https://github.com/flathub/org.kde.skanlite/pull/17#issuecomment-2324484807) using my usb scanner and it works.

Hint: I faced issues when moving from different Flatpak builds (like local/remote), probably due dirty data (like caches) in the app folder. Drop the app folder before to test a different build.
